src/coreclr/inc/clrnt.h

```diff
@@ -510,7 +510,7 @@ RtlVirtualUnwind(
 #define UNW_FLAG_EHANDLER               0x1             /* filter handler */
 #define UNW_FLAG_UHANDLER               0x2             /* unwind handler */
 
-PEXCEPTION_ROUTINE
+PEXCEPTION_ROUTINE inline
```

Comment:
```suggestion
inline PEXCEPTION_ROUTINE
```
Nit: `inline` is typically place before the return type